File: ui/base/events/event_handler.cc
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright (c) 2012 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "ui/base/events/event_handler.h"

namespace ui {

EventHandler::EventHandler() {
}

EventHandler::~EventHandler() {
}

EventResult EventHandler::OnKeyEvent(KeyEvent* event) {
  return ui::ER_UNHANDLED;
}

EventResult EventHandler::OnMouseEvent(MouseEvent* event) {
  return ui::ER_UNHANDLED;
}

EventResult EventHandler::OnScrollEvent(ScrollEvent* event) {
  return ui::ER_UNHANDLED;
}

EventResult EventHandler::OnTouchEvent(TouchEvent* event) {
  return ui::ER_UNHANDLED;
}

EventResult EventHandler::OnGestureEvent(GestureEvent* event) {
  return ui::ER_UNHANDLED;
}

}  // namespace ui

File: ash/webui/recorder_app_ui/resources/core/utils/event_handler.ts
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2024 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

/**
 * An event handler that preventDefault and stopPropagation on the event.
 */
export function suppressEvent(event: Event): void {
  event.stopPropagation();
  event.preventDefault();
}

/**
 * An event handler that stopPropagation on the event.
 */
export function stopPropagation(event: Event): void {
  event.stopPropagation();
}
